schema/annotation-source.yaml

```diff
@@ -161,30 +151,51 @@ $defs:
       A Statement (aka ‘Assertion’) represents a claim of purported truth as made by a particular agent, 
       on a particular occasion.
     heritable_properties:
+      target_proposition:
+        oneOf:
+        - $ref: "#/$defs/Proposition"
+        - $ref_curie: gks.core:CURIE
+        description: The Proposition about which the Statement is made.
+      evidence_direction:
+         type: string
+         enum: [ "supports", "disputes", inconclusive" ] 
+         description: >-
+           A term indicating the net direction of all evidence assessed against the target proposition. 
       evidence_level:
         $ref_curie: gks.core:Coding
         description: >- 
-          A term indicating the overall strength of support for the Statement based on all evidence assessed.
+          A term indicating the overall strength of support for or against the Statement's proposition, 
+          as provided by all evidence evaluated.
```

Comment:
Refine definition to fit updated model / align with VA IM.  Clarifies that evidence strength is assessed specifically against the Statement's _proposition_  (an evidence level/score reflects strength of support for this proposition, not the Statement).